generic/tests/cfg/timedrift.cfg

```diff
@@ -3,6 +3,7 @@
     qemu_stop = off
     i386, x86_64:
         rtc_drift = "slew"
+    ntp_stop_cmd = "(systemctl stop chronyd || service ntpdate stop)"
```

Comment:
window os need be considered too.

Reply:
I don't know if have this problem on x86 platform, @YanhuiMa Could you help test these cases on x86?

Reply:
You mean what problem? For the case, it indeed needs to add ntp_stop_cmd for both x86 and ppc. And the case is not specified only ppc. So yfu and I think cmd related window guest also need to be considered.